simple/src/main/g8/test/Test$name__Camel$.tdml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+$!
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements. See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+!$
+
+<tdml:testSuite suiteName="$name;format="Camel"$" description="$name;format="Camel"$ tests"
+  xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/" 
+  xmlns:fn="http://www.w3.org/2005/xpath-functions" 
+  xmlns:tdml="http://www.ibm.com/xmlns/dfdl/testData" 
+  xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  defaultRoundTrip="onePass">
+
+  <tdml:parserTestCase name="test_$name;format="camel"$_01" root="$name;format="Camel"$" model="$name;format="camel"$.dfdl.xsd">
+    <tdml:document>
+      <tdml:documentPart type="file">/data/test_01.$name;format="camel"$</tdml:documentPart>
+    </tdml:document>
+    <tdml:infoset>
+      <tdml:dfdlInfoset type="file">/xml/test_01.$name;format="camel"$.xml</tdml:dfdlInfoset>
```

Comment:
Should we use file suffixes ".dat" and ".dat.xml" or continue using $name as a file suffix?

I've found there's no need to start the filenames with /, which makes them look like relative filenames and therefore easier to find the data files if one's reading this TDML file.

For the subdirectory names, "data" is fine but "infosets" may be a better name than "xml".